docs/source/api/v3/deliveryservices.rst

```diff
@@ -234,7 +238,9 @@ Response Structure
 		],
 		"maxOriginConnections": 0,
 		"ecsEnabled": false,
-		"rangeSliceBlockSize": null
+		"rangeSliceBlockSize": null,
+                "serviceCategory": null,
+                "serviceCategoryName": null
```

Comment:
Use tabs for indentation.

Code blocks may contain space indents for their contents, but syntactically the indentation to associate the content with the code block must be tabs. So it's easier to just use tabs throughout.